packages/flutter_fuchsia/lib/flutter_fuchsia.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright 2018 The Fuchsia Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Provides an API to test Flutter applications on Fuchsia devices and
+/// emulators.
+///
+/// The application runs in a separate process from the actual test.
```

Comment:
Technically it can run in the same process, so long as it can connect to the application via a web-socket. But we should probably still say that _usually_ it runs in a separate process or even on a separate computer.

Reply:
Updated.